### PR TITLE
Add documentation for disabling presigned url

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

**Review by ozkatz — APPROVED** [review]
THANK YOU

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -114,9 +114,13 @@ This reference uses `.` to denote the nesting of values.
 * `blockstore.gs.credentials_file` `(string : )` - If specified will be used as a file path of the JSON file that contains your Google service account key
 * `blockstore.gs.credentials_json` `(string : )` - If specified will be used as JSON string that contains your Google service account key (when credentials_file is not set)
 * `blockstore.gs.pre_signed_expiry` `(time duration : "15m")` - Expiry of pre-signed URL.
+* `blockstore.gs.disable_pre_signed` `(bool : false)` - Disable use of pre-signed URL.
+* `blockstore.gs.disable_pre_signed_ui` `(bool : false)` - Disable use of pre-signed URL in the UI.
```
Comment (nopcoder) [inline docs/reference/configuration.md:118]: Relevant to all block store that holds this configuration - `diable_pre_signed` needs to `true` to enable this one.

File: docs/reference/configuration.md [hunk 2]
```diff
@@ -114,9 +114,13 @@ This reference uses `.` to denote the nesting of values.
 * `blockstore.gs.credentials_file` `(string : )` - If specified will be used as a file path of the JSON file that contains your Google service account key
 * `blockstore.gs.credentials_json` `(string : )` - If specified will be used as JSON string that contains your Google service account key (when credentials_file is not set)
 * `blockstore.gs.pre_signed_expiry` `(time duration : "15m")` - Expiry of pre-signed URL.
+* `blockstore.gs.disable_pre_signed` `(bool : false)` - Disable use of pre-signed URL.
```
Comment (nopcoder) [inline docs/reference/configuration.md:117]: It is enabled by default on gs

File: docs/reference/configuration.md [hunk 3]
```diff
@@ -132,6 +136,8 @@ This reference uses `.` to denote the nesting of values.
 * `blockstore.s3.server_side_encryption` `(string : )` - Server side encryption format used (Example on AWS using SSE-KMS while passing "aws:kms")
 * `blockstore.s3.server_side_encryption_kms_key_id` `(string : )` - Server side encryption KMS key ID
 * `blockstore.s3.pre_signed_expiry` `(time duration : "15m")` - Expiry of pre-signed URL.
+* `blockstore.s3.disable_pre_signed` `(bool : false)` - Disable use of pre-signed URL.
```
Comment (nopcoder) [inline docs/reference/configuration.md:139]: True by default

File: docs/reference/configuration.md [hunk 4]
```diff
@@ -114,9 +114,13 @@ This reference uses `.` to denote the nesting of values.
 * `blockstore.gs.credentials_file` `(string : )` - If specified will be used as a file path of the JSON file that contains your Google service account key
 * `blockstore.gs.credentials_json` `(string : )` - If specified will be used as JSON string that contains your Google service account key (when credentials_file is not set)
 * `blockstore.gs.pre_signed_expiry` `(time duration : "15m")` - Expiry of pre-signed URL.
+* `blockstore.gs.disable_pre_signed` `(bool : false)` - Disable use of pre-signed URL.
+* `blockstore.gs.disable_pre_signed_ui` `(bool : false)` - Disable use of pre-signed URL in the UI.
 * `blockstore.azure.storage_account` `(string : )` - If specified, will be used as the Azure storage account
 * `blockstore.azure.storage_access_key` `(string : )` - If specified, will be used as the Azure storage access key
 * `blockstore.azure.pre_signed_expiry` `(time duration : "15m")` - Expiry of pre-signed URL.
+* `blockstore.azure.disable_pre_signed` `(bool : false)` - Disable use of pre-signed URL.
```
Comment (nopcoder) [inline docs/reference/configuration.md:122]: True by default

**Review by nopcoder — CHANGES_REQUESTED** [review]

**Review by nopcoder — APPROVED** [review]
LGTM!

**Review by nopcoder — APPROVED** [review]
LGTM!

**Pull request merged**